src/System/Http/Response.php

```diff
@@ -54,19 +54,49 @@ class Response
      */
     public function __construct($content = '', int $respone_code = Response::HTTP_OK, array $headers = [])
     {
-        $headers_content = $content['headers'] ?? [];
         // remove header information
         if (isset($content['headers'])) {
-            unset($content['headers']);
+            foreach ($content['headers'] as $key => $header) {
+                $headers[$key] = $header;
+            }
         }
```

Comment:
i nice to remove content['header'], it will Deprecated but it ok. i will supported

Reply:
okey